README.md

```diff
@@ -71,6 +71,8 @@ The project uses environment variables to manage different configurations. A `.e
 	 - REACT_APP_FIREBASE_APP_ID: Your Firebase App ID.
 	 - REACT_APP_FIREBASE_MEASUREMENT_ID: Your Firebase Measurement ID.
 	 - REACT_APP_DISPLAY_CONSOLE: Handle console logs (`true` or `false`). If left empty, it will be handled as `true`.
+	 - REACT_APP_WEBAUTHN_ORIGIN: WebAuthn origin URL configuration.
+	 - REACT_APP_WEBAUTHN_RPID: WebAuthn relying party ID.
```

Comment:
Actually, `REACT_APP_WEBAUTHN_ORIGIN` is no longer needed after [webauthn credential verification was removed](
https://github.com/wwWallet/wallet-frontend/pull/264/commits/184afa200f053c82a342c23c598d66f8de7ae167#diff-d751f6a6097bc4a258647c44903c897cc1784d33d71736575d93e97cc3292708L136) for offline login. :smile:

We should suggest a suitable default value too:
```suggestion
	 - REACT_APP_WEBAUTHN_RPID: WebAuthn relying party ID (when running locally, set to `localhost`). This must match the `config.webauthn.rp.id` setting in `wallet-backend-server`.
```

Hm, maybe we should automatically fall back to that as a default [in the code](https://github.com/wwWallet/wallet-frontend/pull/264/files#diff-d751f6a6097bc4a258647c44903c897cc1784d33d71736575d93e97cc3292708R1)?

Reply:
I just included those changes to the PR of wallet-frontend

- https://github.com/wwWallet/wallet-frontend/pull/264 https://github.com/wwWallet/wallet-frontend/pull/264/commits/3705a3e154729b7de86e9e2020f070a64ecf7a14